utils/scripts/replay_scenarios.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
-NOT_SUPPPORTED=("K8S_LIB_INJECTION_BASIC" "K8S_LIB_INJECTION_FULL" "TRACE_PROPAGATION_STYLE_W3C" "APM_TRACING_E2E_OTEL" "CROSSED_TRACING_LIBRARIES" "APPSEC_AUTO_EVENTS_RC")
```

Comment:
I guess the K8s_* scenarios won't work on replay mode. We don't launch in the replay mode

Reply:
I'll add a commit for that.